### ISSUE TITLE
Long pressing Notifications back navigation button presents empty navigation list

### ISSUE THREAD

**Issue by dcalhoun** [wordpress-mobile/WordPress-iOS#21769]
### Expected behavior
The navigation menu has a list _with_ titles. 
<img alt="Back navigation list with titles" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/438664/e406e814-cb79-4138-8937-5d68bd08965b" width="300" />

For this particular context, the titles might be: 

* "Notification Details" for the nested notification detail view. 
* "Notifications" for the top-level notifications list view. 

### Actual behavior
The navigation menu has a list _without_ titles. 
<img alt="Back navigation list without titles" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/438664/46f173d6-5656-4209-9cf9-e24aca2dd643" width="300" />

### Steps to reproduce the behavior
1. Navigate to Notifications. 
2. Tap a "like" notification for a post or comment. 
3. Tap the post or comment title for the "like" notification found at the top of the screen content. 
4. Long press the back button. 
5. Without lifting your finder, move your finger down the screen and away from the back navigation menu. 
6. Note the empty list of navigation items.

**Comment by am2089** [comment]
Hello @dcalhoun would I be able to tackle this issue?

**Comment by dcalhoun** [comment]
👋🏻 @am2089. Thank you for inquiring! 

Yes, I believe this to be a good issue to address and we welcome contributions. I will assign this issue to you, but please let me know if you decide you are unable to work on it. Thanks! 🙇🏻

**Comment by am2089** [comment]
@dcalhoun Hey so when I follow the build instructions and input 'git clone git@github.com:wordpress-mobile/WordPress-iOS.git' into the terminal it tells me I don't have the access rights.

**Comment by twstokes** [comment]
👋 Hi @am2089, I wanted to share that [we have a Slack workspace](https://make.wordpress.org/chat/) where it'll be easier for us to assist with project setup. Just drop by the #mobile channel. Thanks for your interest in contributing!

**Comment by am2089** [comment]
Thank you @twstokes I will hop on there.

**Comment by am2089** [comment]
@twstokes Hey, you can take me off this issue so someone else can tackle it. Once I figure out what's going on, on my side, I will see if there is another issue available.

**Comment by twstokes** [comment]
No worries @am2089! Will do.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
